core/src/com/projectkorra/projectkorra/command/PKCommand.java

```diff
@@ -292,7 +292,7 @@ public List<Player> getOnlinePlayers(final CommandSender sender) {
 	 * @return A list of online player names that the sender can see
 	 */
 	public List<String> getOnlinePlayerNames(final CommandSender sender) {
-		return Bukkit.getOnlinePlayers().stream().filter(p -> !(sender instanceof Player) || ((Player) sender).canSee(p)).map(Player::getName).collect(Collectors.toList());
+		return Bukkit.getOnlinePlayers().stream().filter(p -> !(sender instanceof Player player) || player.canSee(p)).map(Player::getName).collect(Collectors.toList());
```

Comment:
In future, avoid doing these unless rewriting code as it just clutters PRs/commit history without actually changing any bytecode

Reply:
here I get it I suppose

Reply:
But here it does serve an actual purpose, List.of() returns the same empty list always instead of using the memory for a new list, its a micro op but one with a purpose

Reply:
Oh weird, love github, that second comment was supposed to be on your comment of "Same with things like this"